guides/k8-akka-cluster/src/main/paradox/index.md

```diff
@@ -0,0 +1,455 @@
+# How to setup Akka cluster on Kubernetes
```

Comment:
This guide describes how to setup Akka services with and without Akka clustering. Maybe we should change the headline than to "How to setup an Akka application on Kubernetes"